### DISCUSSION TITLE
Menus/MenuBar

### DISCUSSION THREAD

**Discussion by roborobs1023** [AllenDang/giu#126] [Q&A] [answered]
Another one. Is it possible to add Menu Items to the right side of the menu bar.

As an example. 

File, Edit, Preferences, Help on the left and on the right side of the window have Account for accessing account information of the user, all within the same menu bar. 

Secondly, is it possible to add icons/images to items in a menu list instead of just text?

**Comment by AllenDang** [comment, marked as answer]
You could use selectable as menu and add context menu to each selectable triggered by left mouse button, and now you could control the layout as what ever you like.

And menu item could be any kind of widget, not just icons/images.

**Reply by roborobs1023** [reply]
Okay.

Thanks.
